docker/docker-compose.yml

```diff
@@ -7,6 +7,9 @@ services:
     build:
       context: ./database
     ports:
-      - "5432:5432"
+      - '5432:5432'
     volumes:
-      - ../.volumes/postgresql:/var/lib/postgresql/data
+      - postgresql:/var/lib/postgresql/data
```

Comment:
You don't need to add this folder in the `.gitignore` ?

Reply:
No this is internal to Docker,  `postgresql` is defined below in the `volumes` section.